src/store/modules/user/actions.js

```diff
@@ -10,6 +10,9 @@ export const loginUser = ({
   cellphone,
   picture_url,
   admin,
+  manager,
```

Comment:
Why are you adding these properties here?

Reply:
It was need to pass the actions.test, our `createUser` has those fields